docs/content/instance-backup/index.md

```diff
@@ -1,38 +1,44 @@
-# Backup instances
-
-### Overview
+## Overview
 
 In Wakame-vdc it is possible to take a running [instance](../jargon-dictionary.md#instance) and turn it into a new [machine image](../jargon-dictionary.md#machine-image). We simply call this process *backup*.
 
 This is a quick and easy way to create new machine images. For example, you could start up an instance, install a web server in it and then back it up. Now you'll have a machine image with a web server installed and you'll immediately be able to start a any amount of web server instances.
 
-### Requirements
+## Requirements
 
 You of course need a working Wakame-vdc installation. You can use either the [VirtualBox demo image](http://wakameusersgroup.org/demo_image.html) or create your own environment using the [installation guide](../installation.md).
 
-You need to tell Wakame-vdc which [backup storage](../jargon-dictionary.md#backup-storage] to use when backing up instances. Both the VirtualBox image and the installation guide will set that up but just in case, here it is again.
+You need to tell Wakame-vdc which [backup storage](../jargon-dictionary.md#backup-storage) to use when backing up instances. Both the VirtualBox image and the installation guide will set that up but just in case, here it is again.
 
 Figure out which backup storage you want to use with `vdc-manage`.
 
-    /opt/axsh/wakame-vdc/dcmgr/bin/vdc-manage backupstorage show
+```bash
+/opt/axsh/wakame-vdc/dcmgr/bin/vdc-manage backupstorage show
+```
 
 On an environment installed according to the installation guide, it will have this output
 
-    UUID            Storage Type         Base URI
-    bkst-local      local                file:///var/lib/wakame-vdc/images/
+```bash
+UUID            Storage Type         Base URI
+bkst-local      local                file:///var/lib/wakame-vdc/images/
+```
 
```

Comment:
This is interesting.  I was wondering what difference the bash would make.  Here it makes the word "local" blue(maybe) because local is a bash keyword.  What should be put here not to color any text?